Draft_no.md

```diff
@@ -3,31 +3,69 @@
 
 > «En veiledning i å vite hva du bør gjøre når ingen forteller deg hva du må gjøre.»
 
-Forord:
+#### Innhold
+
+1 [Om Ducky](#Om-Ducky)  
```

Comment:
To create links to sections use anchors. For example this one is `#1--om-ducky`
You can see the anchors by clicking the links that appear when hovering over the header on the page (https://github.com/DuckyTeam/handbook/blob/drafttomarkdown/Draft_no.md#1--om-ducky)
But as a general rule the anchor will be something like the name of the head with hyphens where the spaces are